src/screens/application/container/applicationContainer.js

```diff
@@ -487,13 +487,6 @@ class ApplicationContainer extends Component {
       const isExistUser = await getExistUser();
 
       realmObject[0].name = currentUsername;
-      dispatch(
-        updateCurrentAccount({
-          name: realmObject[0].username,
-          avatar: realmObject[0].avatar,
-          authType: realmObject[0].authType,
-        }),
-      );
```

Comment:
why did you remove this

Reply:
cuz its not necessary at all we already keeping on redux store